corehq/apps/cloudcare/static/cloudcare/js/debugger/debugger.js

```diff
@@ -1,4 +1,4 @@
-/* globals CodeMirror, gettext, Clipboard */
+/* globals CodeMirror, gettext, Clipboard, FormplayerFrontend */
```

Comment:
I don't think `FormplayerFrontend` is available here for the reasons I mentioned before re: `here()` PR. This is a separate module, does this work?

Reply:
I'm not sure how to explain it because I don't understand the details of what you mean by separate module, but I can confirm from direct testing that the correct value for `tz_offset_millis` definitely gets added in this module and in `webformsession.js`